package.json

```diff
@@ -18,7 +18,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "prettier": "npx prettier . --write"
```

Comment:
Does this install the prettier package every time I run the command? Please change "prettier" command to something like "format" or "format:write"